pkg/client/leaderelection/leaderelection.go

```diff
@@ -21,9 +21,10 @@ limitations under the License.
 // This implementation does not guarantee that only one client is acting as a
 // leader (a.k.a. fencing). A client observes timestamps captured locally to
 // infer the state of the leader election. Thus the implementation is tolerant
-// to arbitrary clock skew, but is not tolerant to arbitrary clock skew rate.
+// to arbitrary clock offset, but is not tolerant to arbitrary clock skew (the
```

Comment:
I think what you had originally is correct.
https://en.wikipedia.org/wiki/Clock_skew#On_a_network
says
"clock skew describes the difference in time shown by the clocks at the different nodes on the network"

I could swear there is an official term for "clock skew rate" but I dug around and couldn't find anything, so I'd just leave as-is for now.